src/diffusers/models/normalization.py

```diff
@@ -205,20 +224,28 @@ def __init__(
         norm_type="layer_norm",
     ):
         super().__init__()
+
         self.silu = nn.SiLU()
         self.linear = nn.Linear(conditioning_embedding_dim, embedding_dim * 2, bias=bias)
         if norm_type == "layer_norm":
             self.norm = LayerNorm(embedding_dim, eps, elementwise_affine, bias)
+        elif norm_type == "fp32_layer_norm":
+            self.norm = FP32LayerNorm(embedding_dim, eps=eps, elementwise_affine=elementwise_affine, bias=bias)
         elif norm_type == "rms_norm":
             self.norm = RMSNorm(embedding_dim, eps, elementwise_affine)
+        elif norm_type == "no_norm":
```

Comment:
I don't think we should call it `AdaLayerNormContinuous` without the normalization layer here 
I think we can move this out and make a special layer in `auraflow_transformers_2d`

Reply:
Done!